app/src/main/java/com/example/workoutapp/WorkoutActivity.kt

```diff
@@ -31,115 +29,44 @@ class WorkoutActivity : ComponentActivity() {
                 Surface(
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colors.background
-                )
-                {
-                    /* Creates a a drop down list containing all the days of the week
-                    * Each day contains a list of exercises done */
-
-                    MyWorkout(
-                        sections = listOf(
-                            CollapsableSection(
-                                title = "Monday",
-                                rows = listOf("Pushups", "Bench Press", " Cardio")
-                            ),
-                            CollapsableSection(
-                                title = "Tuesday",
-                                rows = listOf("Pushups", "Bench Press", " Cardio")
-                            ),
-                            CollapsableSection(
-                                title = "Wednesday",
-                                rows = listOf("Pushups", "Bench Press", " Cardio")
-                            ),
-                            CollapsableSection(
-                                title = "Thursday",
-                                rows = listOf("Pushups", "Bench Press", " Cardio")
-                            ),
-                            CollapsableSection(
-                                title = "Friday",
-                                rows = listOf("Pushups", "Bench Press", " Cardio")
-                            ),
+                ) {
+                    Column {
+                        val workoutweek = listOf(
+                            MyWorkoutDay(
+                                day = "",
+                                exercises = listOf("")
+                            )
                         )
-                    )
+
+                        MyWorkouts(
+                            myWorkoutWeek = workoutweek
+                        )
+                    }
                 }
             }
         }
     }
 }
 
-data class CollapsableSection(val title: String, val rows: List<String>)
+/* Composable function MyWorkouts for displaying a user's workout routine by days of teh week. */
+
+/* The workout routine page is displayed in a multi-row layout using expandable rows that list the user's exercises each day. */
 
-const val MaterialIconDimension = 24f
+/* The user can add exercises to their routines through the plus button at the bottom. */
```

Comment:
Use proper KDoc format above a function.
https://medium.com/@drflakelorenzgerman/documentation-with-kdoc-for-kotlin-android-a93c99dfe74